### PR TITLE
add command to deploy in local and give parameters to serverless invoke

### PR DESCRIPTION
**Pull request by Fabex**
In order to deploy on local machine with serveless I need to give some paremeters to the invoke function

### CODE REVIEW

File: bref [hunk 1]
```diff
@@ -52,6 +52,104 @@ $app->command('init', function (SymfonyStyle $io) {
...
+     * - handler.js
+     * - .bref/**
+     */
+
+    // Run build hooks defined in .bref.yml
+    $buildHooks = $projectConfig['hooks']['build'] ?? [];
+    foreach ($buildHooks as $buildHook) {
+        $io->writeln('Running ' . $buildHook);
+        $commandRunner->run('cd .bref/output && ' . $buildHook);
+    }
```
Comment (mnapoli) [inline bref]: Thanks for the PR! If I'm not mistaken everything above this is duplicated from the `deploy` command? Maybe that code could be shared instead, moved to a class maybe? I'm thinking a `generate()` method in a  `src/Console/Deployer` class for example.

File: bref [hunk 2]
```diff
@@ -52,6 +52,104 @@ $app->command('init', function (SymfonyStyle $io) {
     ]);
 });
 
+$app->command('local [data]', function(?string $data, SymfonyStyle $io) {
```
Comment (mnapoli) [inline bref]: Could you add a comment above this line? Something that says something like "Runs the lambda locally".

File: bref (hunk 1 above)
Comment (Fabex) [reply bref]: Yes you are right, it's a big duplication of deploy command. I would just propose you this feature in a simple way to saw if it's was a good idea.
I will refacto this and send you a new commit

File: bref (hunk 2 above)
Comment (Fabex) [reply bref]: ok no problem

**Comment by mnapoli** [comment]
Thanks, I have built upon this PR (the code wasn't working) and opened #11. I have kept your commit, I'll be closing this one and merging #11.

**Pull request closed without merging**